test/functional/p2p_addr_relay.py

```diff
@@ -179,7 +179,7 @@ def relay_tests(self):
         # of the outbound peer which is often sent before the GETADDR response.
         assert_equal(inbound_peer.num_ipv4_received, 0)
 
-        # Send an empty ADDR message to intialize address relay on this connection.
+        # Send an empty ADDR message to initialize address relay on this connection.
```

Comment:
nit, maybe fix the other outstanding one too:

```bash
$ test/lint/lint-spelling.sh 
src/wallet/test/spend_tests.cpp:57: neccesary ==> necessary
test/functional/p2p_addr_relay.py:182: intialize ==> initialize
```

Reply:
done